examples/Example14/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2021 CERN
+# SPDX-License-Identifier: Apache-2.0
+
+if(NOT TARGET G4HepEm::g4HepEm)
+  message(STATUS "Disabling example14 (needs G4HepEm)")
+  return()
+endif()
+
+if(Geant4_FOUND)
+  if(NOT Geant4_gdml_FOUND)
+    message(STATUS "Disabling example14 (needs Geant4 with GDML support)")
+    return()
+  endif()
+else()
+  message(STATUS "Disabling example14 (needs Geant4)")
+  return()
+endif()
+
+set(sources_g4
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14ActionInitialisation.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14DetectorConstruction.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14DetectorMessenger.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14EMShowerMessenger.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14EMShowerModel.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14EventAction.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14Hit.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14PrimaryGeneratorAction.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14PrimaryGeneratorMessenger.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14RunAction.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Example14SensitiveDetector.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/PhysListHepEm.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/PhysListEmStandard.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/PhysListG4Em.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/PhysicsList.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/PhysicsListMessenger.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/TrackingAction.cc
```

Comment:
No need to qualify this with `${CMAKE_CURRENT_SOURCE_DIR}`, paths are by default relative to the current source directory